PlanAndRide/PlanAndRide.BusinessLogic/Route.cs

```diff
@@ -8,15 +8,146 @@
 
 namespace PlanAndRide.BusinessLogic
 {
-    class Route
+    public class Route
     {
         public string Name { get; set; }
-        public double StartingPosition { get; set; } 
-        public double DestinationPosition { get; set; } 
+        public double StartingPosition { get; set; }
+        public double DestinationPosition { get; set; }
         public string Description { get; set; }
         public bool ShareRoute { get; set; }
         public bool IsPrivate { get; set; }
-        public Review Review { get; set; }
+        public List<Review> Reviews { get; set; }
+
+        public void NewRoute()
+        {
+            //nazwa trasy + sprawdzenie jej wprowadzenia
+
+            Console.WriteLine("Stwórzmy trasę!\n");
+            Console.WriteLine("Nazwa nowej trasy:");
+            Name = Console.ReadLine().Trim();
+
+            while (Name == "")
+            {
+                Console.WriteLine("Wprowadź nazwę nowej trasy:");
+                Name = Console.ReadLine();
+            }
+
+            Console.WriteLine($"\nNazwa nowej trasy to: {Name}\n");
+
+            //opis trasy
+
+            Console.WriteLine("Czy chcesz dodać opis trasy (wprowadź 't' lub 'n': ");
+            var descriptionYesNo = Console.ReadLine().ToLower().Trim();
+
+            do
```

Comment:
jak masz jakąś część kodu, która powtarza się w wielu miejscach to należy zrobić metodę i później z niej korzystać a nie pisać ten sam kod wiele razy :)

Reply:
poprawione. przyznam, że podejrzałem z rozwiązania Rafała ;)